# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Bjørn I. <bjorn.svindseth@online.no>, 2014,2016
# Finn Brudal <finnbrudal@yahoo.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-09-20 18:02+0200\n"
"PO-Revision-Date: 2017-09-21 01:33+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Norwegian Nynorsk (http://www.transifex.com/otf/torproject/language/nn/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: nn\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:101
msgid "You must run this application as root"
msgstr "Du lyt køyra denne applikasjonen som rot"

#: ../tails_installer/creator.py:147
msgid "Extracting live image to the target device..."
msgstr "Dreg ut live-bilete åt måleinheita..."

#: ../tails_installer/creator.py:154
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Skreiv åt einheiti med %(speed)d MB/sec"

#: ../tails_installer/creator.py:184
msgid "Setting up OLPC boot file..."
msgstr "Set upp OLPC-støvelfili..."

#: ../tails_installer/creator.py:321
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Det var vanskeleg å køyra fylgjande ordre: `%(command)s`\nEin meir utførleg feillogg hev vorte skriven åt '%(filename)s'."

#: ../tails_installer/creator.py:340
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Stadfestar SHA1-kontrollsummen åt LiveCD-biletet..."

#: ../tails_installer/creator.py:344
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Stadfestar SHA256-kontrollsummen åt LiveCD-biletet..."

#: ../tails_installer/creator.py:360
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Feil: SHA1'en åt Live-CD'en din er ugyldig. Du kan køyra dette programmet med --noverify-parameteren for å hoppa yver denne stadfestingskontrollen."

#: ../tails_installer/creator.py:366
msgid "Unknown ISO, skipping checksum verification"
msgstr "Ukjend ISO, hoppar yver kontrollsumstadfestingi"

#: ../tails_installer/creator.py:377
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Ikkje nok ledug plass på einheiti.\n%dMB ISO + %dMB yverlegg > %dMB ledug plass"

#: ../tails_installer/creator.py:384
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Skapar %sMB vedvarande yverlegg"

#: ../tails_installer/creator.py:445
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Kan ikkje kopiera %(infile)s åt %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:459
msgid "Removing existing Live OS"
msgstr "Fjernar verande Live-OS"

#: ../tails_installer/creator.py:468 ../tails_installer/creator.py:479
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Kan ikkje chmod'a %(file)s: %(message)s"

#: ../tails_installer/creator.py:472
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Kan ikkje fjerna fil frå fyrre LiveOS: %(message)s"

#: ../tails_installer/creator.py:484
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Kan ikkje fjerna katalog frå fyrre LiveOS: %(message)s"

#: ../tails_installer/creator.py:532
#, python-format
msgid "Cannot find device %s"
msgstr "Finn ikkje eininga %s"

#: ../tails_installer/creator.py:701
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Kan ikkje skriva på %(device)s, hoppar yver."

#: ../tails_installer/creator.py:725
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Nokre inndelingar på måleinheiti %(device)s er monterte. Dei vil verte avmontera fyre innstalleringsgangen tek til."

#: ../tails_installer/creator.py:768 ../tails_installer/creator.py:991
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Ukjend filsystem. Det kan vera at einheiti di treng å verta formatert um."

#: ../tails_installer/creator.py:771 ../tails_installer/creator.py:994
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Ustøtta filsystem: %s"

#: ../tails_installer/creator.py:788
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:792
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Kan ikkje montera einheiti: %(message)s"

#: ../tails_installer/creator.py:797
msgid "No mount points found"
msgstr "Fann ingen monteringspunkt"

#: ../tails_installer/creator.py:808
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "Gjeng inn i unmount_device for '%(device)s'"

#: ../tails_installer/creator.py:818
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Avmonterer monterte filsystem på '%(device)s'"

#: ../tails_installer/creator.py:822
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Avmonterer '%(udi)s' på '%(device)s'"

#: ../tails_installer/creator.py:832
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Montering %s finst etter avmontering"

#: ../tails_installer/creator.py:845
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Delar inn einheit %(device)s"

#: ../tails_installer/creator.py:920
#, python-format
msgid "Updating properties of system partition %(system_partition)s"
msgstr "Uppdaterer eigenskapar åt systeminndelingi: %(system_partition)s"

#: ../tails_installer/creator.py:976
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Ustøtta einheit '%(device)s', ver venleg og meld inn eit insekt."

#: ../tails_installer/creator.py:979
msgid "Trying to continue anyway."
msgstr "Freistar å halda fram likevel."

#: ../tails_installer/creator.py:988 ../tails_installer/creator.py:1381
msgid "Verifying filesystem..."
msgstr "Stadfestar filsystem..."

#: ../tails_installer/creator.py:1012
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Kan ikkje gjera um volumetikett: %(message)s"

#: ../tails_installer/creator.py:1017 ../tails_installer/creator.py:1414
msgid "Installing bootloader..."
msgstr "Installerer støvellastar..."

#: ../tails_installer/creator.py:1043
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr ""

#: ../tails_installer/creator.py:1051 ../tails_installer/creator.py:1432
#, python-format
msgid "Removing %(file)s"
msgstr "Fjernar %(file)s"

#: ../tails_installer/creator.py:1163
#, python-format
msgid "%s already bootable"
msgstr "%s allereie startbar"

#: ../tails_installer/creator.py:1183
msgid "Unable to find partition"
msgstr "Kan ikkje finna inndeling"

#: ../tails_installer/creator.py:1206
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Formaterer %(device)s som FAT32"

#: ../tails_installer/creator.py:1266
msgid "Could not find syslinux' gptmbr.bin"
msgstr ""

#: ../tails_installer/creator.py:1279
#, python-format
msgid "Reading extracted MBR from %s"
msgstr ""

#: ../tails_installer/creator.py:1283
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1296 ../tails_installer/creator.py:1297
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Nullstiller hovudinndelingssektor åt %s"

#: ../tails_installer/creator.py:1302
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Drevet er ei attendekopling, hoppar yver MBR-nullstilling"

#: ../tails_installer/creator.py:1306 ../tails_installer/creator.py:1560
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Reknar ut SHA1'en av %s"

#: ../tails_installer/creator.py:1331
msgid "Synchronizing data on disk..."
msgstr "Skapar samsvar med upplysingar på skivi..."

#: ../tails_installer/creator.py:1373
msgid "Error probing device"
msgstr "Feil ved sondering av einheit"

#: ../tails_installer/creator.py:1375
msgid "Unable to find any supported device"
msgstr "Kan ikkje finna nokor støtta einheit."

#: ../tails_installer/creator.py:1385
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Sjå til at USB-lykelen din er plugga inn og formatert med FAT-filsystemet"

#: ../tails_installer/creator.py:1388
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Ustøtta filsystem: %s\nVer venleg og ta kopi av USB-lykelen din og formater han med FAT-filsystemet."

#: ../tails_installer/creator.py:1455
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Kan ikkje fenga Win32_LogicalDisk; win32com-fyrespurnad gav ingi resultat"

#: ../tails_installer/creator.py:1508
msgid "Cannot find"
msgstr "Finn ikkje"

#: ../tails_installer/creator.py:1509
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Ukjend versjon: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Lastar ned %s "

#: ../tails_installer/gui.py:209
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Feil: Kan ikkje setja etiketten eller fenge UUID'en åt einheita di. Kan ikkje halda fram."

#: ../tails_installer/gui.py:258
#, python-format
msgid "Installation complete! (%s)"
msgstr "Installasjonen er fullført! (%s)"

#: ../tails_installer/gui.py:263
msgid "Tails installation failed!"
msgstr ""

#: ../tails_installer/gui.py:363
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Åtvaring: Dette verktyet treng å køyra som Administrator. For å gjera dette, høgreklikk på ikonet og opn Eigenskapar. Under Kompatibilitet-fanen, kryss av \"Køyr dette programmet som administrator\"-boksen."

#: ../tails_installer/gui.py:375
msgid "Tails Installer"
msgstr ""

#: ../tails_installer/gui.py:454
msgid "Upgrade"
msgstr ""

#: ../tails_installer/gui.py:458 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr ""

#: ../tails_installer/gui.py:465
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:476
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:477
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:516
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:518
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:552
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:562
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Einheiti \"%(pretty_name)s\" er for lita til å installere Tails på (minst %(size)s GB er påkravt)."

#: ../tails_installer/gui.py:584
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:596
msgid "Refreshing releases..."
msgstr "Oppdaterer versjonane "

#: ../tails_installer/gui.py:601
msgid "Releases updated!"
msgstr "Versjonane oppdaterte!"

#: ../tails_installer/gui.py:643
msgid "Installation complete!"
msgstr "Installering fullførd"

#: ../tails_installer/gui.py:644
msgid "Installation was completed."
msgstr ""

#: ../tails_installer/gui.py:693
msgid "Unable to mount device"
msgstr "Kan ikkje montera einheiti"

#: ../tails_installer/gui.py:700 ../tails_installer/gui.py:730
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:701
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:717
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:725
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:726
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:776
msgid "Download complete!"
msgstr "Nedlastinga er ferdig!"

#: ../tails_installer/gui.py:780
msgid "Download failed: "
msgstr "Mislukka nedlasting:"

#: ../tails_installer/gui.py:781
msgid "You can try again to resume your download"
msgstr "Du kan prøva å halda fram med nedlastinga"

#: ../tails_installer/gui.py:789
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Den valde fili er ulesbar. Ver venleg og fiks løyvi hennar eller vel ei onnor fil."

#: ../tails_installer/gui.py:795
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Kan ikkje nytta den valde fili. Du hev kanskje betre hell um du flyttar ISO'en din åt roti av drevet ditt (t.d. C:\\)"

#: ../tails_installer/gui.py:801
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s vald(e)"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr ""

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr ""

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr ""

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr ""

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr ""

#: ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr ""

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""
